conda-recipes/impyla/meta.yaml

```diff
@@ -25,6 +26,7 @@ requirements:
     - thrift_sasl
     - bitarray
     - thriftpy
+    - sasl >=0.2.1
```

Comment:
`sasl` and `thrift_sasl` are soft requirements of impyla -- I didn't see this before, but is there some way to do this with conda builds?